src/main/steps/c100-rebuild/safety-concerns/abduction/child-location/template.njk

```diff
@@ -0,0 +1,20 @@
+{% extends "common/template.njk" %}
+
+{% block page_content %}
+
+    <span class="govuk-caption-l">{{section}}</span>
+    <h1 class="govuk-heading-xl">{{title}}</h1>
+    <div class="govuk-warning-text">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+            <strong class="govuk-warning-text__text">
+                <span class="govuk-warning-text__assistive">{{warning}}</span>
+                   {{warningText}}
+            </strong>
+    </div>
+    <p class="govuk-heading-s">{{safetyConcernsText}}</p>
+    <ul class="govuk-list govuk-list--bullet govuk-!-margin-left-7 govuk-!-margin-bottom-6">
+        <li>{{safetyConcernsBullet1}}</li>
+        <li>{{safetyConcernsBullet2}}</li>
+    </ul>
+
```

Comment:
@ShakiraKP Abduction hyperlinks are missing in your page.
Please include common template.njk file defined in abduction which will add the common links above contact-us. 
```
{% block page_safetyConcerns %}
    {% include "../template.njk"%}
{% endblock %}
```

Reply:
@yogendraU told me to skip these links in the corner and they would be included as a whole story down the line, please confirm whether I should create or do they already exist? @channa-patil

Reply:
@ShakiraKP He might have told you not to separately as we are be having common template.njk with all abduction links. But still you need to include it so that your page shows the required links.
Please include it & push the changes.

Reply:
Completed

Reply:
Completed